### ISSUE TITLE
Is this initiative still active?

### ISSUE THREAD

**Issue by abhishekmaha23** [w3c-ccg/zcap-spec#46]
Hi!
I see that this repo has not seen any update for more than half a year, and some parts of the spec are still incomplete.
Is it still intended to be expanded further at some point, or is it to be considered abandoned?
I'm also curious about the interaction with the capabilityDelegation and capabilityInvocation keys in the DID specification, and if they supersede these specifications.
Also, I find a few references to OCAP-LD, which do not have a draft specification.

**Comment by dlongley** [comment]
Yes, this spec is still active with some implementations in use in various projects. It has taken a backseat while the Verifiable Credentials Data Model 2.0 (and other related specs) are making their way through the standardization process. It will get reprioritized thereafter (there are limited volunteer hours, but there is not limited interest or use).

**Comment by TallTed** [comment]
> the Solid community, whose charter was just accepted as a working group this week

Note that the referenced charter is for the [Linked Web Storage WG](https://www.w3.org/groups/wg/lws/). Solid is only part of the input to their (our) work.

**Comment by gobengo** [comment]
> Is it still intended to be expanded further at some point, or is it to be considered abandoned?

Yes, it is still intended to be expanded further. It is not to be considered abandoned. @dmitrizagidulin and I recently joined as co-editors, and work will continue in the new Capability-based Storage Task Force in W3C Credentials Community Group. Last week was the first of the [biweekly meetings](https://lists.w3.org/Archives/Public/public-credentials/2026Jul/0009.html) of the task force. Join us.

> Also, I find a few references to OCAP-LD, which do not have a draft specification.

I believe these have been resolved. If I'm mistaken, please reopen the issue and point them out.

**Comment by TallTed** [comment]
The [linked message](https://lists.w3.org/Archives/Public/public-credentials/2026Jul/0009.html) says --

> on the 1st and 3rd Thursday of each month.

-- but the [calendar invitation](https://lists.w3.org/Archives/Public/public-credentials/2026Jul/0037.html) says --

> Event is recurring Monthly on Thursday, but only the third instance of this set

-- though it also says --

> twice-a-month (1st and 3rd weeks of the month)

I hope this is sufficient for @dmitrizagidulin and/or @gobengo to fix the calendar details, so these events automagically show up for CCG calendar subscribers.
